manifest.php

```diff
@@ -32,7 +32,7 @@
     'label' => 'Generis Core',
     'description' => 'Core extension, provide the low level framework and an API to manage ontologies',
     'license' => 'GPL-2.0',
-    'version' => '13.1.0',
+    'version' => '13.1.1',
```

Comment:
Until now we were not updating versions and it probably will be outdated until merge to develop/master so I'm not sure we should update now